src/tests/Microsoft.PowerFx.Core.Tests/ExpressionErrorTests.cs

```diff
@@ -171,7 +171,7 @@ public void Empty()
             // We don't want null IEnumerables. 
             // Null gets normalized to empty.
             var internalErrors = (IEnumerable<IDocumentError>)null;
-            var errors = ExpressionError.New(internalErrors);
+            var errors = ExpressionError.New(internalErrors, CultureInfo.CurrentCulture);
```

Comment:
>CultureInfo.CurrentCulture [](http://example.com/codeflow?start=61&length=26)

"avoid CultureInfo.CurrentCulture"
Does it also apply here?

Reply:
This one is intentional otherwise there is no difference with line 178